### ISSUE TITLE
add a "replace" action button

### ISSUE THREAD

**Issue by ldt** [pqina/vue-filepond#188]
**Is your feature request related to a problem? Please describe.**
I would like to use filepond in an admin page where the user has to upload a file (a synonyms list) and can only replace it.
I can't find how to simply tell the user s.he can re-drag a file or clic to update the file that was previously uploaded.

**Describe the solution you'd like**
I would love an icon, like the X for remove, to invite the user to update the file and trigger a replace upload action

**Describe alternatives you've considered**
I've considered tweaking the css to always display the idle-label which I would change depending if there is an uploaded file or not.

**Comment by rikschennink** [comment]
Hi, this sounds like a question that should be asked on Stack Overflow.

To give a short answer. You can load an existing file and offer remove options, with the original file removed you can add a new one, by setting `allowReplace` to `true` you can also drop new files over existing files.

**Comment by ldt** [comment]
Hi, Thanks for your quick reply.
I already used `allowReplace`.
My point is that the user will not know s.he can update the file unless s.he removes the existing one as the `idle-label` has disappeared.
So I ended up by adding another label below stating "To replace the existing one, drop a file or **browse**". I test `pond.getFile()` to check if this label should be displayed or not and `pond.browse()` if the user clicks on it.

I would have preferred a specific icon on the existing file to quickly invite the user to replace it.
